java/spacewalk-java.spec

```diff
@@ -1,7 +1,7 @@
 #
 # spec file for package spacewalk-java
 #
-# Copyright (c) 2019 SUSE LINUX GmbH, Nuernberg, Germany.
```

Comment:
Please note that I excluded changing the order of the dependencies, as the changes were huge, we have a lot of `%if` in this package, and I do not want to risk breaking GMC or Uyuni 2021.06.